frameworks/hdfs/tests/test_auth.py

```diff
@@ -13,6 +13,8 @@
 
 
 log = logging.getLogger(__name__)
+foldered_name = sdk_utils.get_foldered_name(config.SERVICE_NAME)
```

Comment:
I would suggest that we not use global variables, and instead use test fixtures if required. I don't feel too strongly about this though.